component/file_cache/file_cache.go

```diff
@@ -697,8 +699,33 @@ func (fc *FileCache) OpenFile(options internal.OpenFileOptions) (*handlemap.Hand
 		// If the lock is held then we cannot update the file
 		f, err = os.OpenFile(localPath, os.O_WRONLY, options.Mode)
 		if err != nil {
-			log.Err("FileCache::OpenFile : Failed to open file %s in WR mode", localPath)
-			return nil, err
+			if os.IsPermission(err) {
```

Comment:
You mentioned we should figure out a more efficient way to deal with this problem right? Could you create an item for it?

Reply:
https://msazure.visualstudio.com/One/_backlogs/backlog/XStore-DevExp/Epics/?showParents=true&workitem=13735363